common/es/src/main/java/io/apiman/common/es/util/DefaultEsClientFactory.java

```diff
@@ -198,12 +195,13 @@ private void waitForElasticsearch(RestHighLevelClient client, long pollingTime)
                             // measure time if health request is successful
                             final Date endTime = new Date();
                             long pollingTimeMeasure = endTime.getTime() - startTime.getTime();
-                            logger.info("Took "+ pollingTimeMeasure + " milliseconds for polling Elasticsearch");
-
+                            logger.info("Took " + pollingTimeMeasure + " milliseconds for polling Elasticsearch");
                             // wake up the waiting thread
                             cdl.countDown();
                         }
-
+                    } catch (ElasticsearchException | SSLException e) {
+                        logger.error("Error connecting to Elasticsearch: ", e);
+                        exception.setValue(e);
                     } catch (IOException e) {
                         logger.info("Unable to reach Elasticsearch. Will continue polling.");
```

Comment:
In the logger let's at least print out the exception message to guide us, let's say:

```
logger.info("Unable to reach Elasticsearch with error ({0}). Will continue polling.", e.getMessage())
```

Reply:
Will test everything, was just a draft pr to test if CI fails :)